csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -186,18 +207,42 @@ public void RegisterCustomOpLibrary(string libraryPath)
         /// managed by the user (created using the CreateTensorWithDataAsOrtValue API) and it must outlive the session object
         /// to which it is added.
         /// </summary>
-        public void AddInitializer(string name, OrtValue ort_value)
+        public void AddInitializer(string name, OrtValue ortValue)
         {
-            NativeApiStatus.VerifySuccess(NativeMethods.OrtAddInitializer(handle, name, ort_value.Handle));
+            NativeApiStatus.VerifySuccess(NativeMethods.OrtAddInitializer(handle, name, ortValue.Handle));
```

Comment:
>name [](start = 82, length = 4)

Ditto. UTF-8

Reply:
See above - let us perform a purge across the C# codebase in a different PR (this wasn't even added in my PR)